### ISSUE TITLE
Add text to image based analyses

### ISSUE THREAD

**Issue by ErnstBay** [mostaphaRoudsari/honeybee#654]
I was looking to add text to radiance images. See below for an example. 
http://www.solemma.net/img/Features/Animations/GlareSweep.gif

Mostapha suggested the following:
You can use Radiance's psign (http://radsite.lbl.gov/radiance/man_html/psign.1.html).

He also said that creating a new component for this purpose would make sense.
Check the glare component for an example.
https://github.com/mostaphaRoudsari/honeybee/blob/5d1689daaadf6b3bb4bb7a721bf7ab25d05e261f/src/Honeybee_Glare%20Analysis.py#L316

**Comment by mostaphaRoudsari** [comment]
@devngc are you interested to implement this. It is very similar to what you have already done for the Glare component.

**Comment by devngc** [comment]
@mostaphaRoudsari,
Yes. I would be happy to take this up. Thank you! Sorry for the delay in responding. Something's wrong. I have stopped receiving Github notifications in Email. Luckily I stumbled upon this when I created a new issue myself. I am looking into it.

**Comment by devngc** [comment]
@mostaphaRoudsari,
I have started working on this one. Do you really think there should be a new component for adding text to the TIFF? I can also add a new input to the existing **Honeybee_Convert HDR to TIF**  component.

**Comment by chriswmackey** [comment]
@devngc ,
This seems like something that is better addressed with just a new input and not a whole new component.  Adding it to the "Convert HDR to TIF" and the other image converters sounds good.

**Comment by devngc** [comment]
@chriswmackey,
Thank you for your feedback. @mostaphaRoudsari has a different opinion about this. He will share his views here soon.

**Comment by TheodoreGalanos** [comment]
Hi everyone,

Speaking completely selfishly, I would REALLY not mind a component that can add text to an image. If the component is able to do this to existing images even better (although I'm not sure if this would be possible). 

However, it's also not clear to me if something like this is related and/or beneficial to the general scope and specific capabilities of Ladybug Tools.

I'll be quietly keeping an eye on how this goes! Thanks!

Regards,
Theodore.

**Comment by mostaphaRoudsari** [comment]
I think having it as a separate component will be quite useful. It can be an optional addition to the convertors too but it's really a separate step and you can do a lot with this new functionality that I think it deserves it's own component.

**Comment by devngc** [comment]
@mostaphaRoudsari,
Thank you for sharing your views. I will move ahead in the direction of creating a new component. Will post here as soon as I have something.

**Comment by chriswmackey** [comment]
Hello all,

I stand corrected and there's clearly a consensus here in favor of a separate component. Now that I see how psign works and I realize that there are several useful inputs, I agree we should implement it as a separate component. My only question is whether we think this component can take an image in any format and add the text (gif, Tiff, HDR) or does the image need to be in a specific format.

**Comment by mostaphaRoudsari** [comment]
@chriswmackey, the Radiance functionality only works for HDR format but we can always use `image > TIFF > HDR` workflow for non-HDR images

**Comment by chriswmackey** [comment]
Ok. All sounds good!

**Comment by hamzajh19** [comment]
Hi, was this issue sorted in the end?
